draft-ietf-httpbis-digest-headers.md

```diff
@@ -732,6 +774,14 @@ integrity at transport layer that protects HTTP header fields.
 
 A `Digest` header field using NOT RECOMMENDED digest-algorithms SHOULD NOT be used in signatures.
 
+## Usage with PATCH
+
+A `200 OK` response to a PATCH request would contain
+the `Digest` of the patched item, and the etag of the new object.
```

Comment:
What about other status codes? Perhaps we can give an example that failed and that returns the Digest and etag of the unpatch item (if that is what happens).

Reply:
moved to PR

Reply:
Further reasoning moved to a new issue #958